README.md

```diff
@@ -15,9 +15,9 @@ This library can be easily integrated with any authentication gem ([devise](http
 
 ## Requirements
 
-* Rails >= 3.2
-* ActiveRecord >= 3.2 <b>or</b> Mongoid >= 3.1
-* supports ruby 2.0/1.9.3, JRuby 1.6.0+ (in 1.9 mode) and Rubinius 2.0.0dev (in 1.9 mode)
+* Rails >= 4.2
+* ActiveRecord >= 4.2 <b>or</b> Mongoid >= 4.0
+* supports ruby 2.2+, JRuby 1.6.0+ (in 1.9 mode) and Rubinius 2.0.0dev (in 1.9 mode)
```

Comment:
I'll update this but we likely want to drop the JRuby, Rubinius mentions or update them along with the next line about Ruby 1.8 as it's irrelevant since we say it's a 2.2+ lib.